controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -300,6 +301,77 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) error {
 		}
 		// Else: nothing for eBPF at the moment
 	}
+
+	// Connection tracking stage
+	endTimeout, err := time.ParseDuration(b.desired.Processor.ConnectionEndTimeout)
+	if err != nil {
+		return err
+	}
+	updateInterval, err := time.ParseDuration(b.desired.Processor.ConnectionUpdateInterval)
+	if err != nil {
+		return err
+	}
+	lastStage = lastStage.ConnTrack("extract_conntrack", api.ConnTrack{
+		KeyDefinition: api.KeyDefinition{
+			FieldGroups: []api.FieldGroup{
+				{Name: "src", Fields: []string{"SrcAddr", "SrcPort"}},
+				{Name: "dst", Fields: []string{"DstAddr", "DstPort"}},
+				{Name: "common", Fields: []string{"Proto"}},
+			},
+			Hash: api.ConnTrackHash{
+				FieldGroupRefs: []string{
+					"common",
+				},
+				FieldGroupARef: "src",
+				FieldGroupBRef: "dst",
+			},
+		},
+		OutputRecordTypes: []string{
+			"flowLog",
+			"newConnection",
+			"updateConnection",
+			"endConnection",
+		},
+		OutputFields: []api.OutputField{
+			{
+				Name:      "Bytes",
+				Operation: "sum",
+			},
+			{
+				Name:      "Bytes",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+			{
+				Name:      "Packets",
+				Operation: "sum",
+			},
+			{
+				Name:      "Packets",
+				Operation: "sum",
+				SplitAB:   true,
+			},
+			{
+				Name:      "numFlowLogs",
+				Operation: "count",
+			},
+			{
+				Name:      "TimeFlowStartMs",
+				Operation: "min",
+			},
+			{
+				Name:      "TimeFlowEndMs",
+				Operation: "max",
+			},
+		},
+		EndConnectionTimeout: api.Duration{
+			Duration: endTimeout,
+		},
+		UpdateConnectionInterval: api.Duration{
+			Duration: updateInterval,
+		},
```

Comment:
Once https://github.com/netobserv/flowlogs-pipeline/pull/357 is merged, this snippet should be change to something like:
```go
	Scheduling: []api.ConnTrackSchedulingGroup{
		{
			Selector:                 nil, // Default group. Match all flowlogs
			UpdateConnectionInterval: api.Duration{Duration: endTimeout},
			EndConnectionTimeout:     api.Duration{Duration: updateInterval},
		},
	},
```

Reply:
Thanks for the info @ronensc !

Reply:
Is there a plan to create a new FLP release for this ? @jotak 

https://github.com/netobserv/network-observability-operator/pull/252/files#diff-c761638f65b982f9ecf3717616920a5a3c0bb1670fd8ee7e9ab51afc4f166581R378

https://github.com/netobserv/network-observability-operator/pull/252/files#diff-c761638f65b982f9ecf3717616920a5a3c0bb1670fd8ee7e9ab51afc4f166581R407

Reply:
https://github.com/netobserv/network-observability-operator/pull/252/commits/decd18365a3da2a23ff09c8a0b8be65c0c208e23